src/screens/HomeScreen.js

```diff
@@ -147,6 +147,7 @@ const renderItem = ({ item }) => {
       {...{
         buttonTitle,
         fetchPolicy,
+        isHomeScreen: query === QUERY_TYPES.EVENT_RECORDS,
```

Comment:
why is the prop called `isHomeScreen` when the check is if the query is event records?
all the components can be used on other screens as well, where we cannot have a prop `isHomeScreen` that we probably set to true or false.
can you think of a different naming solution?
what would you describe it to me in words? in which case do the prop needs to be true?

Reply:
I chose this option because this problem only occurs with the event list on `HomeScreen`, because there is a special case for the Event list in `listHook`.

How about changing the name to `isEventListOnHomeScreen`?

Reply:
then it is still home screen. what is that "special case"? can it be named? which data is the reason?

Reply:
We have 2 different event listings. The 1st list is the list we list on the `EventList` screen. The data in this list starts at index 1. Because index 0 is `SectionHeader`.

The other event list is the one we show in `HomeScreen`. The data in this list starts at index 0.

We need to make the image of the first data bigger and the images of the other data smaller. Therefore, we need to consider 2 different situations in the event list.

In `listHook`, we solved the problem in `EventList` screen by adding filter according to index, but this time the list view in `HomeScreen` is broken. So we needed to add a special control for the event list in `HomeScreen`.

Reply:
You're absolutely right, by the way. I don't want to name a prop that is used everywhere as `HomeScreen` either, but unfortunately I haven't found a better solution.

Reply:
> We have 2 different event listings. The 1st list is the list we list on the EventList screen. The data in this list starts at index 1. Because index 0 is SectionHeader.
>
> The other event list is the one we show in HomeScreen. The data in this list starts at index 0.

thanks for the explanation. this lets me think about `isIndexStartingAt0` or `isIndexStartingAt1` for example?

Reply:
Logical! I updated the code with `isIndexStartingAt0`.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/7a5d494d9248f9cef1385dab1e4a0cc85376f69a